docs/import/create_score_asset.js

```diff
@@ -95,24 +135,35 @@ function convertToNumber(value) {
 
 /**
  * Post-processes the join of snap data and tiger geometries to form a single
- * feature.
+ * feature with SNAP data (including percentage).
  * @param {ee.Feature} feature
  * @param {string} snapKey
  * @param {string} totalKey
  * @return {ee.Feature}
  */
 function combineWithSnap(feature, snapKey, totalKey) {
   const snapFeature = ee.Feature(feature.get('primary'));
+  const snapPop = convertToNumber(snapFeature.get(snapKey));
+  const totalPop = convertToNumber(ee.Number.parse(snapFeature.get(totalKey)));
+  const badData = null;
+  const snapPercentage = ee.Algorithms.If(
+      totalPop,
+      ee.Algorithms.If(
+          ee.Algorithms.IsEqual(snapPop, null), badData,
```

Comment:
why using badData in place of null while also using null?

Reply:
It was my attempt to be a little more document-y: the null in the isEqual is being compared with snapPop. badData is a value that we return. Would a more descriptive name like `nullSnapPercentage` help instead of badData, or is this just a badIdea?

Reply:
Gotcha, no I think this is fine just confirming I wasn't missing anything else going on